src/features/contactmoment/service.ts

```diff
@@ -68,6 +68,7 @@ export function useKlantContactmomenten(
     url.searchParams.append("extend[]", "contactmoment.medewerker");
     url.searchParams.append("extend[]", "x-commongateway-metadata.owner");
     url.searchParams.append("extend[]", "contactmoment.todo");
+    url.searchParams.append("limit", "10");
```

Comment:
I noticed we weren't paginating it at all, this seems more user friendly.

Reply:
Fixed